tests/phpunit/api/v4/UnitTestCase.php

```diff
@@ -0,0 +1,241 @@
+<?php
+
+namespace api\v4;
+
+use api\v4\Traits\TestDataLoaderTrait;
+use Civi\Test\HeadlessInterface;
+use Civi\Test\TransactionalInterface;
+
+/**
+ * @group headless
+ */
+class UnitTestCase extends \PHPUnit_Framework_TestCase implements HeadlessInterface, TransactionalInterface {
```

Comment:
@totten for phpunit6 compatibility

```suggestion
class UnitTestCase extends \PHPUnit\Framework\TestCase implements HeadlessInterface, TransactionalInterface {
```

Reply:
@seamuslee001 will that also work on phpunit5?

Reply:
I believe so yes @colemanw